downstairs/src/region.rs

```diff
@@ -1765,9 +1776,14 @@ impl Region {
                     &self.log,
                 )
                 .await
-            };
+            }?;
+
+            if extent.dirty().await {
```

Comment:
Could this be inside the else?
Like, if we are creating an extent, it would seem strange if it was already dirty :)

It's probably not going to make a noticeable difference in startup time though.

Reply:
it could be, yeah